pygmsh/built_in/geometry.py

```diff
@@ -270,13 +270,16 @@ def __init__(
             line_loop, plane_surface
             )
 
+    # pylint: disable=too-many-branches
     def extrude(
             self,
             input_entity,
             translation_axis=None,
             rotation_axis=None,
             point_on_axis=None,
             angle=None,
+            layers=False,
```

Comment:
`layers` is a number, right? If yes, the variable should be called `num_layers`, and the default should be `None`. Instead of `if layers` below, one should test for `if layers is [not] None`.